### PR TITLE
rfc-9, fixes #115, adding MultiDevFailed data type

### LINKED ISSUE #115: Add MultiDevFailed to RFC-9

### PR DESCRIPTION
**Pull request by piogor**

### CODE REVIEW

File: 9/DataTypes.md [hunk 1]
```diff
@@ -326,3 +326,25 @@ Results of failed operations (exceptions) within the Tango Controls are sent as
...
+```
+
+* Any value of MultiDevFailed type SHALL follow \<milti-dev-failed-value\> rule:
+```abnf
+ multi-dev-failed-value = 1*named-dev-error 
+
+ named-dev-error = name index-in-call 1*error
+
+ name = *VCHAR
+ index-in-call = num-val ; of lon data type in range of -(2^31) to (2^31)-1
```
Comment (bourtemb) [inline 9/DataTypes.md]:
```suggestion
 index-in-call = num-val ; of long data type in range of -(2^31) to (2^31)-1
```

File: 9/DataTypes.md [hunk 2]
```diff
@@ -326,3 +326,25 @@ Results of failed operations (exceptions) within the Tango Controls are sent as
  desc = *VCHAR ; SHOULD describe the failure in more details
  origin = *VCHAR ; SHALL identify the operation or the tango object which caused the failure, eg. function name
 ```
+
+* MultiDevFailed type name is as follows:
+```abnf
+ MultiDevFailed = "MultiDevFailed"
+```
+
+* Any value of MultiDevFailed type SHALL follow \<milti-dev-failed-value\> rule:
```
Comment (bourtemb) [inline 9/DataTypes.md]:
```suggestion
* Any value of MultiDevFailed type SHALL follow \<multi-dev-failed-value\> rule:
```

File: 9/DataTypes.md [hunk 3]
```diff
@@ -326,3 +326,25 @@ Results of failed operations (exceptions) within the Tango Controls are sent as
...
+```abnf
+ multi-dev-failed-value = 1*named-dev-error 
+
+ named-dev-error = name index-in-call 1*error
+
+ name = *VCHAR
+ index-in-call = num-val ; of long data type in range of -(2^31) to (2^31)-1
```
Comment (bourtemb) [inline 9/DataTypes.md]: Indeed, an index being a long as specified by the IDL. But I guess we will get some troubles if people start using negative indices?

Comment (piogor) [reply 9/DataTypes.md]: Should I change it to 0 - ...?

Comment (bourtemb) [reply 9/DataTypes.md]: Good question! I don't know what's the  best.
-(2^31) to (2^31)-1 is the corresponding range for the long type but I guess in this specific case, we won't use (and probably don't want to use) negative values?

Comment (piogor) [reply 9/DataTypes.md]: Then I suggest this:
```suggestion
 index-in-call = num-val ; of long data type (32 bits) in the range of 0 to (2^31)-1
```

Comment (bourtemb) [reply 9/DataTypes.md]: @piogor, fine with me.

**Pull request merged**